python/ray/serve/_private/long_poll.py

```diff
@@ -398,10 +408,18 @@ def notify_changed(
         object_key: KeyType,
         updated_object: Any,
     ):
-        self.snapshot_ids[object_key] += 1
+        try:
+            self.snapshot_ids[object_key] += 1
+        except KeyError:
+            # Initial snapshot id must be >= 0, so that the long poll client
+            # can send a negative initial snapshot id to get a fast update.
+            # They should also be randomized to try to avoid situations where,
+            # if the controller restarts and a client has a now-invalid snapshot id
+            # that happens to match what the controller restarts with,
+            # the client wouldn't receive an update.
```

Comment:
Instead of explaining the edge case here, could you link my comment?

Reply:
Done!